Mathlib/CategoryTheory/Comma/Final.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2024 Jakob von Raumer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob von Raumer
+-/
+import Mathlib.CategoryTheory.Functor.KanExtension.Adjunction
+import Mathlib.CategoryTheory.Limits.Final
+import Mathlib.CategoryTheory.Grothendieck
+
+/-!
+# Finality of Projections in Comma Categories
+-/
+
+universe v₁ v₂ v₃ u₁ u₂ u₃
+
+namespace CategoryTheory
+
+namespace Comma
+
+open Limits Functor CostructuredArrow
+
+variable {A : Type v₁} [Category.{v₁} A]
+variable {B : Type v₁} [Category.{v₁} B]
+variable {T : Type v₁} [Category.{v₁} T]
```

Comment:
As discussed, please check how much work it would be to generalize the universes here using `AsSmall` and either implement it or leave a comment here somewhere that the universes can be generalized.